scripts/api/setup-datasetfields.sh

```diff
@@ -7,3 +7,4 @@ curl http://localhost:8080/api/admin/datasetfield/load -X POST --data-binary @da
 curl http://localhost:8080/api/admin/datasetfield/load -X POST --data-binary @data/metadatablocks/astrophysics.tsv -H "Content-type: text/tab-separated-values"
 curl http://localhost:8080/api/admin/datasetfield/load -X POST --data-binary @data/metadatablocks/biomedical.tsv -H "Content-type: text/tab-separated-values"
 curl http://localhost:8080/api/admin/datasetfield/load -X POST --data-binary @data/metadatablocks/journals.tsv -H "Content-type: text/tab-separated-values"
+curl http://localhost:8080/api/admin/datasetfield/load -X POST --data-binary @data/metadatablocks/codemeta.tsv -H "Content-type: text/tab-separated-values"
```

Comment:
**suggestion:** We might want to discuss to make this schema experimental, as we changed plans for #8812, too.

Reply:
Yes, good thought. Once we merge #8812 we could even follow up with a small PR to add Codemeta to the list of experimental blocks (in the Appendix of the User Guide, that is) and link here, if you want. That is, we could go ahead and advertise the work being done before putting the tsv itself into the code base. Same with other blocks being worked on: Darwin Core, 3D Object, etc. The idea is that people shouldn't start from scratch.

Reply:
I merged the branch with latest `develop` and as #8812 is now merged, I moved the block to experimental, too.